### DISCUSSION TITLE
🫡 S2 2.0 版本 roadmap

### DISCUSSION THREAD

**Discussion by lcx-seima** [antvis/S2#2174] [Announcements]
去年 11 月发布 2.0-alpha（next） 版本后，收获了不少社区的关注。
也有信赖 S2 的同学已经在生产环境部署了 next 版本（😅 功能待稳定，暂不推荐）。
为方便大家了解进度，经过内部盘点，确定了未来几个月正式版的投入事项和 roadmap。

<img width="548" alt="CleanShot 2023-04-04 at 11 48 34@2x" src="https://user-images.githubusercontent.com/6716092/229682466-a8fd742a-03ba-4daf-acda-289e3bfdb8e9.png">

如有疑问，可回复讨论。再次感谢大家的关注和支持 🫡

**Comment by G0wtherZ** [comment]
2.0 版本有定好发布日期了吗？

**Comment by lijinke666** [comment]
[🎉 S2 2.0 版本开始内测啦 !!! ](https://github.com/antvis/S2/discussions/1933)
